web_src/css/actions.css

```diff
@@ -57,3 +57,25 @@
   gap: .25rem;
   align-items: center;
 }
+
+.run-list .flex-item-trailing {
+  flex-wrap: nowrap;
+}
+
+.run-list-ref {
+  display: inline-block !important;
```

Comment:
Why this overriding? IIRC `ui label` is `inline-block` or `inline-flex` by default?

Reply:
It's `inline-flex` and that does not work with `text-overflow: ellipsis`.

Reply:
I think we could introduce a general approach , instead of patching the styles for more cases:

### demo

```
<div class="ui label" style="width: 100px;"><span class="gt-ellipsis" style="min-width: 0;">Loooooooooooooooooooooooooooooong</span></div>
```


### suggestion

1. add `min-width: 0;` to `gt-ellipsis`
2. use `<div class="ui label"><span class="gt-ellipsis">..........</span></div>
3. add a devteset ui sample

What do you think?

Reply:
If `min-width: 0` reliably makes the ellipsis work, then yes it seems like a good solution. Personally I've never understood exactly how `min-width: 0` works.

Reply:
https://css-tricks.com/flexbox-truncated-text/

https://stackoverflow.com/questions/26465745/ellipsis-in-flexbox-container


![image](https://github.com/go-gitea/gitea/assets/2114189/2bd72ef9-3c61-4bde-ab36-4e08e883c5a3)

Reply:
Ooops, I just found that something I wrote is wrong. I meant that "min-width" should be added to "flex" elements. 

Sorry for the misleading.

Reply:
So that `min-width` on `gt-ellipsis` will only work of that element is also the flex child, which is not always the case. Still I suppose it won't hurt to have the rule.